study-builder/studies/prion/i18n/en.conf

```diff
@@ -241,6 +241,7 @@
 
     //Medical questionnaire
     "medicalName": "Medical Questionnaire",
+    "medicalSecondName": """Medical Questionnaire Update ${ddp.activityInstanceNumberDisplay(1, 2, "#")""",
```

Comment:
I recommend putting the space into the `prefix` argument, so that the second instance name will render cleaning without a trailing space. Also watch out to close the curly braces. Something like:
```
Medical Questionnaire Update${ddp.activityInstanceNumberDisplay(1, 2, " #")}
```